### PR TITLE
update(Hotwiring): New Minigame

### PR DESCRIPTION
**Pull request by Savage912-Official**
Added new minigame to hotwiring that uses ox_lib and updated ox_lib to have the minigame.

New Minigame: https://i.imgur.com/PN8Pt1z.png

### CODE REVIEW

File: resources/usa_vehtheft/cl_hotwire.lua [hunk 1]
```diff
@@ -148,15 +148,15 @@ AddEventHandler('veh:toggleEngine', function(_hasKey, _engineOn)
         while GetGameTimer() - beginTime < 3000 do
           Citizen.Wait(0)
           ShowHelp('You do not have the keys to this vehicle!', 0)
-          exports.globals:notify('You do not have the keys to this vehicle!')
+          --exports.globals:notify('You do not have the keys to this vehicle!')
```
Comment (minipunch) [inline resources/usa_vehtheft/cl_hotwire.lua:151]: Why remove this?

Comment (Savage912-Official) [reply resources/usa_vehtheft/cl_hotwire.lua:151]: I just didn’t see the need of having two different ones.

Comment (minipunch) [reply resources/usa_vehtheft/cl_hotwire.lua:151]: I think this one might be good to keep cause the `ShowHelp` one appears under the chat box in the upper left which can be annoying

and the other appears in the bottom left so it won't be blocked

Comment (Savage912-Official) [reply resources/usa_vehtheft/cl_hotwire.lua:151]: Alright

Comment (Savage912-Official) [reply resources/usa_vehtheft/cl_hotwire.lua:151]: If you wanna change it go for it if not I will when I’m home this weekend

Comment (Savage912-Official) [reply resources/usa_vehtheft/cl_hotwire.lua]: Updated This

**Comment by minipunch** [comment]
Why did this need to be updated? Or how is this new one better?

**Review by minipunch — APPROVED** [review]

**Comment by minipunch** [comment]
Didn't test it so u better hope u tested it good enough lul

**Pull request merged**